cloud/aws/templates/aws_oidc/variables.tf

```diff
@@ -578,3 +578,8 @@ variable "delete_automated_db_backups" {
   type        = bool
   default     = false
 }
+variable "auto_confirm_sns_subscription" {
```

Comment:
Just a FYI, for this to be settable via the civiform_config.sh file, we would also need to add an entry to cloud/aws/templates/aws_oidc/variable_definitions.json:

```
  "AUTO_CONFIRM_SNS_SUBSCRIPTION": {
    "required": false,
    "secret": false,
    "tfvar": true,
    "type": "bool"
  }  
```